tests/run_examples.py

```diff
@@ -302,4 +298,29 @@ def run_all(nodist=False, nostream=False):
 
 
 if __name__ == "__main__":
-    sys.exit(run_all("nodist" in sys.argv or not __has_dist__, "nostream" in sys.argv))
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--nodist",
+        action="store_true",
+        default=False,
+        help="Skip examples with distributed execution",
+    )
+    parser.add_argument(
+        "--nostream",
+        action="store_true",
+        default=False,
+        help="Skip examples with data streaming",
+    )
+    parser.add_argument(
+        "--nosklearnex",
+        action="store_true",
+        default=False,
+        help="Skip sklearnex examples",
+    )
```

Comment:
When we are using this new argunment?

Reply:
We should use it in conda recipe building to fix the problem discussed in the link from the description.